modules/ding_frontend/ding_frontend.install

```diff
@@ -174,3 +174,16 @@ function ding_frontend_update_7002(&$sandbox) {
 function ding_frontend_update_7003() {
   module_enable(array('manualcrop'));
 }
+
+/**
+ * Add translations for search page elements.
+ */
+function ding_frontend_update_7004() {
+  ding_frontend_locale_add_translation(
+    'Sorted by',
+    'da',
+    'Sortéret efter'
+  );
+
+  ding_frontend_locale_clear_cache();
```

Comment:
Translation updates should be done in the `.po` files and executed using `ding2_update_N()` hook calling `ding2_translation_update()`.